app/components/sidebars/main/main_sidebar.js

```diff
@@ -139,16 +139,13 @@ export default class ChannelSidebar extends Component {
             drawerOpened: false,
         });
         this.resetDrawer();
+        Keyboard.dismiss();
     };
 
     handleDrawerOpen = () => {
         this.setState({
             drawerOpened: true,
         });
-
-        if (this.state.openDrawerOffset !== 0) {
-            Keyboard.dismiss();
```

Comment:
Do you have screenshot or screen capture on landscape view?

Reply:
![aug-09-2018 17-47-19](https://user-images.githubusercontent.com/4973621/43898394-518f28ec-9bfc-11e8-90e5-28a06eead1c8.gif)

Is there anything, in particular, I have to look for?

Reply:
Oh thanks, I think that particular code is added in case of landscape view.  Looks great :)